core/templates/dev/head/pages/community-dashboard-page/services/question-suggestion.service.ts

```diff
@@ -37,6 +38,7 @@ angular.module('oppia').factory('QuestionSuggestionService', [
           }
         };
         $http.post(url, data);
+        onSuccess();
```

Comment:
This should be inside the promise right?
`$http.post(url, data).then(onSuccess);`